domain/alexandrekc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "AlexandreKc",
+    "email": "sebastiancarrera2021@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://alexandrekc.github.io/PortafolioPersonal/"
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, or anything combination of those. See docs for more information: https://docs.is-a.dev/